gammapy/image/maps.py

```diff
@@ -260,8 +260,8 @@ def solid_angle(self):
         """
         Solid angle image
         """
-        xsky, ysky = self.coordinates(mode='edges')
-        omega = -np.diff(xsky, axis=1)[1:, :] * np.diff(ysky, axis=0)[:, 1:]
+        coord = self.coordinates(mode='edges')
+        omega = -np.diff(coord.l, axis=1)[1:, :] * np.diff(coord.b, axis=0)[:, 1:]
```

Comment:
I think the way to access lon / lat on a SkyCoord in any frame is via `coord.data.lon` and `coord.data.lat`. Would that work here?

```
In [2]: from astropy.coordinates import SkyCoord

In [3]: SkyCoord(42, 43, unit='deg')
Out[3]: 
<SkyCoord (ICRS): (ra, dec) in deg
    (42.0, 43.0)>

In [4]: s = SkyCoord(42, 43, unit='deg')

In [5]: s.l
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-5-bb9361e12b88> in <module>()
----> 1 s.l

/Users/deil/software/anaconda/lib/python3.4/site-packages/astropy/coordinates/sky_coordinate.py in __getattr__(self, attr)
    434         # Fail
    435         raise AttributeError("'{0}' object has no attribute '{1}'"
--> 436                              .format(self.__class__.__name__, attr))
    437 
    438     def __setattr__(self, attr, val):

AttributeError: 'SkyCoord' object has no attribute 'l'

In [6]: s.ra
Out[6]: <Longitude 42.0 deg>

In [7]: s.data
Out[7]: 
<UnitSphericalRepresentation (lon, lat) in deg
    (42.0, 43.0)>

In [8]: s.data.lon
Out[8]: <Longitude 42.0 deg>

In [9]: s.galactic.data.lon
Out[9]: <Longitude 2.5230336177186996 rad>
```